client/src/features/transactions/TransactionsToolbar.css

```
@@ -0,0 +1,43 @@
+}
+
+.transactions-toolbar-button {
+  display: inline-flex;
+  align-items: center;
+  gap: var(--spacing-xs);
+  padding: var(--spacing-xs) var(--spacing-s);
+  border: none;
+  border-radius: var(--spacing-xs);
+  background: var(--color-medium);
+  color: var(--color-white);
+  font: 700 15px/normal 'Quicksand', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Quicksand" (font-family-name-quotes)

(font-family-name-quotes)